### ISSUE TITLE
Hide Authorization header credentials in verbose mode

### ISSUE THREAD

**Issue by mikedunker** [jakubroztocil/httpie#335]
When the request headers are being echoed back, I'd like the default behavior to be to **** out the credentials.

Instead of this being displayed:
`Authorization: Basic aHR0cGllOmlzQXdlc29tZQ==`
I'd like to see:
`Authorization: Basic ************************`

Similarly:
`Authorization: Bearer *******************************`

Maybe a new option (like --show-creds) could be used to show the credentials.

**Comment by jakubroztocil** [comment]
Good idea. People accidentally share sensitive data like this all the time. Thinking out loud:

* Having this on by default goes slightly against having the output verbatim.
* `Authorization` isn't the only header with sensitive values out there (think all the `Token`, `X-API-Token`, etc.).

**Comment by ncharaf** [comment]
Hello,

Any update regarding this issue ? 

Is it possible to implement something similar to what is done on [httpx](https://github.com/encode/httpx/blob/master/httpx/_utils.py#L151-L160)

Thx
